contrib/guix/README.md

```diff
@@ -212,7 +212,7 @@ details.
   Override the reference UNIX timestamp used for bit-for-bit reproducibility,
   the variable name conforms to [standard][r12e/source-date-epoch].
 
-  _(defaults to the output of `$(git log --format=%at -1)`)_
+  _(defaults to the output of `git -c log.showSignature=false log --format=%at -1`)_
```

Comment:
https://github.com/monero-project/monero/blob/1179269478f9b615d4b550c38b9ba81c77901aec/contrib/guix/guix-build#L208

https://github.com/monero-project/monero/blob/1179269478f9b615d4b550c38b9ba81c77901aec/contrib/guix/libexec/build.sh#L283

nit for grep.